Components/Core/Inc/Command.hpp

```diff
@@ -48,11 +46,16 @@ class Command
     //~Command();    // We can't handle memory like this, since the object would be 'destroyed' after copying to the RTOS queue
 
     // Functions
-    uint8_t* AllocateData(uint16_t dataSize);    // Dynamically allocates data for the command
-    bool CopyDataToCommand(uint8_t* dataSrc, uint16_t size);    // Copies the data into the command, into newly allocated memory
-    bool SetCommandToStaticExternalBuffer(uint8_t* existingPtr, uint16_t size);    // Set data pointer to a pre-allocated buffer, if bFreeMemory is set to true, responsibility for freeing memory will fall on Command
+    uint8_t* AllocateData(uint16_t dataSize);  // Dynamically allocates data for the command
+    bool CopyDataToCommand(uint8_t* dataSrc,
+                           uint16_t size);  // Copies the data into the command, into newly allocated memory
```

Comment:
Change this to one line if possible.

Maybe note down what clang format option this is, if possible, because it's looks quite bad with how we've previously done comments.

Reply:
Again this is just due to the column limit. We might want to rethink our commenting structure or we disable the column limit.